doc/nofoma.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- This is an automatically generated file. -->
```

Comment:
This file should not be added to the repository, no `title.xml`. Please remove both files from the PR (I've now updated `.gitignore` on `main` to exclude these files)